draft-ietf-tls-subcerts.md

```diff
@@ -345,20 +346,20 @@ valid_time:
 
 expected_cert_verify_algorithm:
 
-: The signature algorithm of the credential key pair, where the type
+: The signature algorithm of the Credential key pair, where the type
   SignatureScheme is as defined in {{RFC8446}}. This is expected to be
   the same as CertificateVerify.algorithm sent by the server.  Only signature
-  algorithms allowed for use in CertificateVerify messages are allowed.  When
-  using RSA, the public key MUST NOT use the rsaEncryption OID, as a result,
+  algorithms allowed for use in Certificate Verify messages are allowed.  When
+  using RSA, the public key MUST NOT use the rsaEncryption OID. As a result,
   the following algorithms are not allowed for use with delegated credentials:
   rsa_pss_rsae_sha256, rsa_pss_rsae_sha384, rsa_pss_rsae_sha512.
 
 ASN1_subjectPublicKeyInfo:
 
-: The credential's public key, a DER-encoded {{X.690}} SubjectPublicKeyInfo as defined in
+: The Credential's public key, a DER-encoded {{X.690}} SubjectPublicKeyInfo as defined in
 {{RFC5280}}.
 
-The delegated credential has the following structure:
+The Delegated Credential has the following structure:
```

Comment:
For consistency, we should say "DelegatedCredential" instead.

Reply:
👍